src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios/02 Advanced Examples.md

```diff
@@ -3,11 +3,22 @@ title: 'Advanced Examples'
 excerpt: 'Advanced Examples using the k6 Scenario API - Using multiple scenarios, different environment variables and tags per scenario.'
 ---
 
-## Using multiple scenarios
+You can use multiple scenarios in one script, and these scenarios can be run in sequence or in parallel.
+So besides helping you model workloads, scenarios can also help you make more flexible and composable test logic.
+Some ways that you can use scenarios include the following:
+- Sequence workloads
+- Add per-scenario tags and environment variables
+- Make scenario-specific thresholds.
+- Run scenarios in different VU [lifecycle functions](/using-k6/test-lifecycle),
 
-This configuration will first execute a scenario where 50 VUs will try to run as many iterations
-as possible for 30 seconds. It will then transition to the next scenario, executing 100 iterations
-per VU for a maximum duration of 1 minute.
+
+## Sequence multiple scenarios
```

Comment:
The problem with this (and with "Sequence workloads" and "flexible and composable test logic" above) is that it's not actually that easy to run scenarios "in sequence" :disappointed: We don't have a `startAfter` logic for scenarios (and we probably won't ever have it), just `startTime`... Which is not sequencing, it's a time offset

Related k6 issue: https://github.com/grafana/k6/issues/1342

Reply:
Oh, I thought I posted responses to this and the lines comment. I guess I deleted them when I did my own review :confused: 

With `maxDuration` can't you specify sequences? I see the limitations: you have to set "time thresholds" for the scenario, and you can't configure scenarios to immediately run one after the other, but you can sequence. Maybe we should just state this limitation (if I'm not mistaken with how it works).

Reply:
Ah, I see that not every scenario has a max duration.

Reply:
I tried to fix this with [18f5b76](https://github.com/grafana/k6-docs/pull/936/commits/18f5b767ce34218ff39bf40279e144659eb82dc7). I don't want to overstate or give incorrect information.

If I still am overstating sequencing, let's just delete those lines. But it does seem like the the arrival rate executors have set durations. So I don't understand why you couldn't set one executor to last 5 minutes and another to start after 5 minutes.

Reply:
It is mostly fine now, I think, I just made a tiny change suggestion inline